src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -514,7 +514,6 @@ export default class AirportModel {
         .done((data) => {
             try {
                 // eslint-disable-next-line no-undef
-                log('Parsing terrain');
```

Comment:
@erikquinn if you're all right with it going away that is completely ok with me.  I hate seeing those logs. 💣